### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by ArieHein**
### Describe Your Changes

Christmas is early and you get the first present in the shape of spelling fixes.
Sorry for the big amount :)

### Checklist

- [x] My change adheres [VictoriaMetrics contributing guidelines](https://docs.victoriametrics.com/contributing/).

### CODE REVIEW

**Comment by jiekun** [comment]
Wow, it looks like there are a lot of typo fixes here! Thanks for contributing. 
May I ask if this was checked and corrected by some tools?

**Comment by ArieHein** [comment]
You're welcome and yes I'm using cspell directly from my VSCode. with basic dictionaries
and that's not a lot. On the weekend, i have found about 700 files with at least one spell issue on the general azure docs repo :)
...i know...everyone has strange hobbies :)  )

I know that you are using cspell as well and have set it up in the make file using a container,
Im just not sure its working correctly or at least not acting as a gate to block

I assume for a PR like mine that has only *.md files, it might not actually do any spelling test as the PR clause in main.yml expects at least one *.go file to be in the PR.

File: docs/helm/README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Get the pods lists by running these commands:
 ```console
 kubectl get pods -A | grep 'victoria-metrics'
 
-# or list all resorces of victoria-metrics
+# or list all resources of victoria-metrics
```
Comment (hagen1778) [inline docs/helm/README.md:95]: @AndrewChubatiuk should this changes be made to helm repo?

File: docs/operator/CHANGELOG.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ aliases:
 - [vmsingle/vlogs](https://docs.victoriametrics.com/operator/resources): makes better compatible with argo-cd by adding ownerReference to PersistentVolumeClaim. See this [issue](https://github.com/VictoriaMetrics/operator/issues/1091) for details.
 - [operator](https://docs.victoriametrics.com/operator/): reduces reconcile latency. See this [commit](2a9d09d0131cc10a0f9e32f0e2e054687ada78f7) for details.
 - [operator](https://docs.victoriametrics.com/operator/): reduces load on kubernetes api-server. See this commits: [commit-0](a0145b8a89dd5bb9051f8d4359b6a70c1d1a95ce), [commit-1](e2fbbd3e37146670f656d700ad0f64b2c299b0a0), [commit-2](184ba19a5f1d10dc2ac1bf018b2729f64e2a8c25).
-- [operator](https://docs.victoriametrics.com/operator/): enables client cache back for `secrets` and `configmaps`. Adds new flag `-controller.disableCacheFor=seccret,configmap` to disable it if needed.
+- [operator](https://docs.victoriametrics.com/operator/): enables client cache back for `secrets` and `configmaps`. Adds new flag `-controller.disableCacheFor=secret,configmap` to disable it if needed.
```
Comment (hagen1778) [inline docs/operator/CHANGELOG.md:54]: @AndrewChubatiuk should this changes be made to operator repo?

**Review by hagen1778 — APPROVED** [review]
Thanks!

**Comment by hagen1778** [comment]
> Wow, it looks like there are a lot of typo fixes here! Thanks for contributing.

@jiekun we have https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/cspell#spell-checking but it needs to be run manually. It is not part of CI because it is slow.

**Pull request merged**